tools/script_template.py

```diff
@@ -1,5 +1,43 @@
 import argparse
 import yaml
+import alara_output_processing as aop
+import openmc
+import numpy as np
+
+def write_out_tables(run_out, run_lbl, time_unit):
+    '''
+    Takes the output of a single ALARA run and converts output tables to Pandas DataFrame
+    inputs: 
+        run_out (path to ALARA output file)
+        run_lbl (name of the run)
+        time_unit (units for cooling time)
+    output: output_tables (Pandas DataFrame containing all output data)
+    '''
+    alara_data = aop.FileParser(run_out, run_lbl, time_unit)
+    output_tables = alara_data.extract_tables()
+    return output_tables
+
+def open_flux_file(flux_file):
+    flux_lines = open(flux_file, 'r').readlines()
+    return flux_lines
+
+def parse_flux_lines(flux_lines):
+    '''
+    Uses provided list of flux lines and group structure applied to the run to create an array of flux entries, with:
+    # rows = # flux entries / # group structure bins
+    # columns = # group structure bins
+    input : flux_lines (list of lines from ALARA flux file)
+    output : flux_array (numpy array of shape # zones/intervals/mixtures x )
```

Comment:
I think ALARA uses one flux spectrum per interval.  It may be true that there is only one interval per zone, and therefore it could be zones... but simpler to just refer to intervals.
```suggestion
    output : flux_array (numpy array of shape # intervals x number of energy groups)
```